packages/core/contracts/bot-helpers/action-wrappers/LiquidationWithdrawer.sol

```diff
@@ -1,5 +1,6 @@
-pragma solidity ^0.6.0;
-pragma experimental ABIEncoderV2;
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+pragma abicoder v2;
```

Comment:
nit: any reason to keep these? Might be nice to just completely drop them given that it's the default now (could be confusing that files that _don't_ have this pragma still have the v2 encoder enabled anyway).

Reply:
I mainly left it like this just to be as explicit as possible but we can remove it to be more concise. I don't really mind either way.